src/tests/test_common/homestore_test_common.hpp

```diff
@@ -174,7 +177,21 @@ class HSTestHelper {
         uint64_t min_chunk_size{0};
         vdev_size_type_t vdev_size_type{vdev_size_type_t::VDEV_SIZE_STATIC};
     };
-
+#ifdef _PRERELEASE
+    flip::FlipClient m_fc{HomeStoreFlip::instance()};
+#endif
+#ifdef _PRERELEASE
+    void set_flip_point(const std::string flip_name) {
+        flip::FlipCondition null_cond;
+        flip::FlipFrequency freq;
+        freq.set_count(10000);
+        freq.set_percent(100);
+        m_fc.inject_noreturn_flip(flip_name, {null_cond}, freq);
+        auto hsi = HomeStore::instance();
+        hsi->set_flip_point(flip_name);
```

Comment:
as noted above, its better to directly set flip points like done in other UT's.

Reply:
done